router/binders_test.go

```diff
@@ -364,3 +364,61 @@ func TestRuntimeValidateResponseSchema(t *testing.T) {
 		})
 	}
 }
+
+func TestRequestBodyWithContentLengthFactory(t *testing.T) {
```

Comment:
Can you add two more cases this test?
1. A body that fails validation.
2. The same as above but with a content type in the skip list so it will not fail.